src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -396,10 +396,15 @@ bool AutoDownloadRule::matchesSmartEpisodeFilter(const QString &articleTitle) co
             m_dataPtr->lastComputedEpisodes.append(episodeStr + u"-REPACK");
             m_dataPtr->lastComputedEpisodes.append(episodeStr + u"-PROPER");
         }
+
+        return true;
```

Comment:
This is the only thing that needs to be added. Could you revert all the other changes?